serverless/serverless
# Move packaging to framework level

## What did you implement:

Restores compatibility with provider plugins. See #3344 

## How did you implement it:

The packaging is now moved to the framework level. The global package plugin overwrites the `package:createDeploymentArtifacts`.

## How can we verify it:

Use "serverless deploy" with different plugins (including AWS). The command should work as before.
Additionally check if deploy and package still work for the AWS plugin with a given package option.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
